admin-site/src/ErrorMessages.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  formEmptyNameField : `Please enter a label/name above`
```

Comment:
Please remove 'label/', let's just use 'name'

Reply:
Same for the label on the actual field? I put both cause in user terms it's "name", but we have it as label in developer terms (in the code).